docs/content/gds/admin/configuration/gds.en.md

```diff
@@ -1,7 +1,159 @@
+| GDS_CERTMAN_REQUEST_INTERVAL    | duration | 10m               | The interval between certificate request processing runs                                                       |
+| GDS_CERTMAN_REISSUANCE_INTERVAL | duration | 24h               | The interval between certificate reissuance processing runs                                                    |
+| GDS_CERTMAN_STORAGE             | string   |                   | The path to a directory on disk where CertMan temporarily downloads certificates (otherwise a tmp dir is used) |
+| GDS_DIRECTORY_ID                | string   | vaspdirectory.net | (Reused) The network ID the GDS serves, either vaspdirectory.net or trisatest.net (or .dev)                    |
+| SECTIGO_USERNAME                | string   |                   | The username to authenticate with the Sectigo API                                                              |
+| SECTIGO_PASSWORD                | string   |                   | The password to authenticate with the Sectigo API                                                              |
+| SECTIGO_PROFILE                 | string   |                   | The certificate profile to use (see below for details)                                                         |
+| SECTIGO_ENDPOINT                | string   |                   | The endpoint to connect to Sectigo on (leave blank for production)                                             |
+| SECTIGO_TESTING                 | bool     |                   | If Sectigo is in testing mode it will not make actual Sectigo requests                                         |
+
+Note that CertMan also needs valid email (SendGrid) and Google Secrets configurations if it is enabled.
+
+The Sectigo username, password, and profile are required if the Sectigo config is not in testing mode. The profile must be a valid certificate profile, which can be either "CipherTrace EE" for the TestNet or "CipherTrace End Entity Certificate" for the MainNet. These profiles determine what parameters must be sent to Sectigo to make certificate requests, and what is populated on the certificate (including what intermediate authority is used) when it is generated.
+
+Do not set the `SECTIGO_ENDPOINT` unless you're in testing, development, or staging mode and are pointing it to a local cathy server.
+
+### Backups
+
+The Backup manager is only available when the GDS is using the LevelDB database store. It is a background process that clones the database into a zipped folder on disk that can be downloaded for backup purposes.
+
+| EnvVar              | Type     | Default | Description                                                      |
+|---------------------|----------|---------|------------------------------------------------------------------|
+| GDS_BACKUP_ENABLED  | bool     | false   | If true, enables the backup background process.                  |
```

Comment:
The description sentence here ends in a `.` while others don't. This is a consistency nitpick and may be ignored.